src/test/java/picard/analysis/TheoreticalSensitivityTest.java

```diff
@@ -207,22 +206,182 @@ public void testHetSensDistributions() throws Exception {
     }
 
     @Test(dataProvider = "hetSensDataProvider")
-    public void testHetSensTargeted(final double expected, final File metricsFile) throws Exception{
+    public void testHetSensTargeted(final double expected, final File metricsFile) throws Exception {
         final double tolerance = 0.000_000_01;
 
         final MetricsFile Metrics = new MetricsFile();
         Metrics.read(new FileReader(metricsFile));
```

Comment:
not your code...(perhaps...) but should try-with-resources

Reply:
fixed